src/rpc/types.rs

```diff
@@ -23,9 +23,34 @@ pub struct Status {
     // pub throughput: f64,
 }
 
+#[derive(Debug, Clone, Default, PartialEq)]
+pub enum RouteGroup {
+    #[default]
+    Ethereum,
+    StrataCL,
+}
+
+impl RouteGroup {
+    pub fn from_method_name(method_name: &str) -> Self {
+        if method_name.starts_with("alp_") {
+            Self::StrataCL
+        } else {
+            Self::Ethereum
+        }
+    }
+
+    pub fn from_config(group: Option<&str>) -> Self {
+        match group {
+            Some("strata") => Self::StrataCL,
+            _ => Self::Ethereum,
+        }
+    }
```

Comment:
Just make this be a `FromStr` impl and support failing to parse.  If someone passed `"solana"` it wouldn't make sense to return `Ethereum`.